CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## [1.2.2](https://github.com/ably/ably-flutter/tree/v1.2.2)
+
+This release adds support for push notifications, including device activation with Ably (including device registration with APNs / FCM), subscribing devices for push notifications and handling the push notifications in your Flutter application, without writing iOS or Android code. Check out the dedicated documentation (`PushNotifications.md`) and the example app (`push_notification_service.dart`) for more information. We also fixed some bugs that were either reported in Github issues or found internally.
```

Comment:
Links are broken.

Reply:
Oh, do you mean, for example https://github.com/ably/ably-flutter/compare/v1.2.2-preview.1...v1.2.2 ?

I haven't created the tags yet, and will do that after this PR is merged. I am following https://github.com/ably/ably-flutter/blob/main/CONTRIBUTING.md

Reply:
No. You have links in brackets in text you added in latest commit. The text prior to them is missing square bracket counterparts.

Reply:
I'm not sure what you mean. Here is the commit that you refer to, but I cannot see anything wrong to fix: https://github.com/ably/ably-flutter/blob/b823b812fc95714c6670797a50bfcef059e1c588/CHANGELOG.md

Reply:
I was also thinking 
 (`PushNotifications.md`) and the example app (`push_notification_service.dart`) 
Are these not supposed to be links?

Reply:
In being pragmatic I chose **not** to make them links, since this is a low visibility page and this information is more properly documented elsewhere. But this got me thinking this might be a low quality (and duplicate) piece of information. Duplicating information and linking directly to github might result in broken links. So I simplified the sentence, and moved some parts to the documentation instead, referencing a local path instead of a github URL: cf909e4b6d233fae32bdebe47bc2f3bcbbd7e15a

Reply:
I kept the link to the push notifications documentation, I think it is actually helpful: 5dac4ea6df7b2ff0821a1c45341698b75f6f5b2c

Reply:
So perhaps @QuintinWillison meant `(`PushNotifications.md`)` as a link? My intention there was not to make it a link, which is why I was confused, since there were no links in that line.

Reply:
Why would you not make it a link? Same goes for the example app. These can safely be absolute links.